packages/story-editor/src/app/rightClickMenu/provider.js

```diff
@@ -775,6 +782,18 @@ function RightClickMenuProvider({ children }) {
     ]
   );
 
+  const canTranscodeResource = useCallback(
+    (resource) => {
+      const { id, isExternal } = resource || {};
```

Comment:
```suggestion
      const { id, isExternal } = resource;
```

Where would `resource` be `null`?

Reply:
It is called with a condiontal, `element?.resource`, meaning it could be nulll

Reply:
That optional chaining usage is wrong though, there's no need for it.

Check the code in question:

`canTranscodeResource` is only ever called when `isVideo` is true, which is only the case when `selectedElement` is not `null`.

https://github.com/google/web-stories-wp/blob/ca85940900e608639796e5d9833017109a1b409e/packages/story-editor/src/app/rightClickMenu/provider.js#L953-L962

That means we can remove that `?` from that call, and remove the `resource || {}`.

(Aside, instead of `(resource) => { const { id, isExternal } = resource || {};` one could just do `(resource ={}) => {
      const { id, isExternal } = resource;` anyway)

Reply:
After changing this, I get this error. 

should contain enabled "trim video"
<img width="1084" alt="Screenshot 2021-11-29 at 14 18 33" src="https://user-images.githubusercontent.com/237508/143884169-5c106562-2274-44ba-9331-27fb82a5750f.png">

Reply:
Also you are wrong here. 
> (resource) => { const { id, isExternal } = resource || {}; one could just do (resource ={}) => { const { id, isExternal } = resource;

There are the same. 

`(resource ={})` sets a default value. If null is passed, the default value will not be set. 
```
const { id, isExternal } = resource || {};
```
Ensure that if resource is null, it falls back to an empty object.

Reply:
> After changing this, I get this error.

That's odd 🤔